### PR TITLE
Merge, edits, corrections on #118, #119, #120

### PR DESCRIPTION
**Pull request by jlevy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -898,6 +907,21 @@ time to IPO has been increasing, and was **7.7 years** in **2016** compared to *
 🚧 What are the restrictions and regulations on selling stock that affect employees at
```
Comment (RacheltheEditor) [inline README.md]: @jlevy let me know if you did want me to draft up this section in its own PR or elsewhere.

File: README.md [hunk 2]
```diff
@@ -898,6 +907,21 @@ time to IPO has been increasing, and was **7.7 years** in **2016** compared to *
...
+shareholders can only make educated guesses about the likely future value, at a time when
+they will be able to sell stock.
+After all, private company stock is simply a legal agreement that entitles you to
+something of highly uncertain value, and could well be worthless in the future, or highly
+valuable, depending on the fate of the company.
+
+☝️ We’ll discuss the notion of a company officially assigning a
+[fair market value](#409a-valuations) later, but even if a company gives you a value for your
+stock for tax and accounting purposes, it doesn’t mean you can expect to sell it for that
+value!
```
Comment (RacheltheEditor) [inline README.md]: Great improvements!

File: README.md [hunk 3]
```diff
@@ -1061,20 +1088,28 @@ they are not *employee* stock options.
...
 purchase your stock.
 
+🚧 Note this is not the same as ESPPs.
+
 🄳 The **strike price** is the fixed price per share at which stock options can be
 exercised, as set in an employee’s stock option agreement.
 The strike price is generally set lower (often much lower) than what people expect will be
 the future value of the stock, which means you can make money when you sell the stock.
 
+🄳 Using stock options to purchase stock at the strike price is called **exercising** the
```
Comment (RacheltheEditor) [inline README.md]: 👍

File: README.md [hunk 4]
```diff
@@ -745,32 +767,37 @@ Here are a few ways you might hear of or see in documents:
...
-    common stock allocated at a company’s formation.
+[**preferred stock**](https://en.wikipedia.org/wiki/Preferred_stock). In general, preferred stock
+is stock that has rights, preferences, and privileges that common stock does not have.
+Typically, investors get preferred stock, and founders and employees get common stock (or
+an [option](#stock-options) to buy common stock).
+
+The exact number of classes of stock and the differences can vary company to company, and,
+in a startup, at [each round of funding](#stages-of-a-startup). If “rights, preferences, and
+privileges” sounds like a confusing legal expression,
+[it is](https://www.americanbar.org/publications/blt/2014/01/04_bigler.html).
```
[the remark this answers is not in the document]
Comment (jlevy) [reply README.md]: Softened a little. But not going to say "ask HR," as typically HR would not be able to share such things.

File: README.md (hunk 1 above)
Comment (jlevy) [reply README.md]: Feel free to put a draft paragraph in Notion, if you have it; but let's focus on production PRs now.

Comment (RacheltheEditor) [reply README.md]: 👍

**Comment by jlevy** [comment]
Thanks @RacheltheEditor ; I've made a couple other changes just you can post-merge review. Merging these.

**Comment by RacheltheEditor** [comment]
Great!

**Pull request merged**